pkg/bgpv1/test/testdata/svc-traffic-policy.txtar

```diff
@@ -1,7 +1,9 @@
 #! --test-peering-ips=10.99.4.101,10.99.4.102
 
 # Tests advertisements with external / internal service traffic policy.
-# Service routes should be withdrawn for a service with local traffic policy in case of no local endpoints.
+# LoadBalancer IPs are always advertised regardless of externalTrafficPolicy and local endpoints,
+# as they are externally-facing addresses meant for external reachability.
```

Comment:
This statement is not correct, externalTrafficPolicy must be respected for LoadBalancer IPs, it was designed specifically for this. For example see https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip